sdk/eventhub/Azure.Messaging.EventHubs/src/EventHubsConnectionStringProperties.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.ComponentModel;
+using System.Runtime.CompilerServices;
```

Comment:
I think this as a mistake, but I don't want to kick off the Core pipeline again, so I'll see if I can remove locally and let it ride along elsewhere.